install-dev/upgrade/sql/1.7.8.6.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode='';
+SET NAMES 'utf8mb4';
+
+ALTER TABLE `PREFIX_state` MODIFY COLUMN `name` VARCHAR(80) NOT NULL;
```

Comment:
You probably should delete this file, it's not included in PrestaShop anymore since v8